TestingScripts/Coverage/CoverageOpenCPP.py

```diff
@@ -0,0 +1,767 @@
+#---------------------------------------------------------------------------------------------
+#  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+#  See COPYRIGHT.md in the repository root for full copyright notice.
+#---------------------------------------------------------------------------------------------
+import xml.etree.ElementTree as ET
+import os, sys
+import subprocess
+import datetime
+import time
+import argparse
+
+#Common Scripts to be used by any task
+scriptsDir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+commonScriptsDir = os.path.join(scriptsDir, 'CommonTasks')
+sys.path.append(commonScriptsDir)
+
+from CodeCoverage import CodeCoverage
+import Components as cmp
+
+#-------------------------------------------------------------------------------------------
+# Class to manipulate Excel Reporting
+# bsiclass
+#-------------------------------------------------------------------------------------------
+class ExcelReport:
+    def __init__(self, fileName):
+        import xlsxwriter
```

Comment:
Here we are using a third party python package. Do I have to include it's name somehere like in some license files or anywhere else?

Reply:
If you have already mentioned it in the documentation than that should be enough.